src/main/webapp/i18n/en/participation.json

```diff
@@ -2,18 +2,18 @@
     "artemisApp": {
         "participation": {
```

Comment:
> I think we indeed meant the participation (german: Teilnahme), which is associated with submissions, a result, etc. When reading "Delete Participant", I would assume, that we delete a participating person or team, but actually, we are deleting the object "Participation" with the associated submission, result, etc. and do not change the participant(s). Is the word Participation wrong here, I saw that the related commit message indicated a "misuse of the word participation" but, after my explanation, is it really a misuse?

@FreshLlamanade what about the general comment I left? Should we reconfirm the use of "Participation" here?

Reply:
I would also argue that participation (Teilnahme) is correct. At least this is what DeepL tells me as appropriate translation

Reply:
Ah, I forgot to respond to that. I agree with what you wrote there and acknowledge that "participant" does not always capture what was meant by "Teilnahme". The trouble is, there is simply no good way to use the word "participation" in that way. I have to hold firm that it is not quantifiable; you can't have "a" participation or multiple "participations" in English. I am trying to be as polite as possible when I say that it made me cringe the first time I saw it used that way.
That said, in cases like "Delete Participant" I find myself doubting whether the change was really necessary in the first place. "Delete Participation" is fine; I think I got ahead of myself and failed to see that it was not actually quantified there. I will go through these changes and revert what I can.

Reply:
>  I am trying to be as polite as possible when I say that it made me cringe the first time I saw it used that way. 😆 

Maybe we could just avoid the plural and only use `Participation`. 
Imagine we replace the marked occurrences with `Participation`, IMO it would be ok, even if an overview of several teams/students or individual participation (e.g. more than more than one) is displayed.
<img width="1560" alt="Bildschirmfoto 2021-05-27 um 21 10 32" src="https://user-images.githubusercontent.com/41366522/119883850-48769b00-bf30-11eb-9ec4-a13596152e68.png">

Reply:
I think that would surely put us on a better track than before. "8 Participation" sounds like more of a mistake than just using the incorrect plural, though. In that case I really think "8 Participants" communicates the same information.

Reply:
ah true, I did not see the number. We could also just not show the number here (the total sum of entries in this list is shown at the bottom anyways). Or in this case use participants, indeed. I'm afraid that has to be decided on a case-by-case basis.

Reply:
Please keep the number 😅

Reply:
Feels like I've opened Pandora's box 😄

Reply:
DeepL tells me there is a plural **participations**

I also found:
https://simple.wiktionary.org/wiki/participations

Maybe it's just not a typical term used in some English speaking countries?

Reply:
I don't know what else to say than DeepL is incorrect 😅 I guarantee that participation is quite strongly uncountable. It sounds absurd to me and my American peers

Reply:
I've pushed a commit to try and find a middle ground in a few cases